index.html

```diff
@@ -54,14 +54,80 @@ <h1>find your movie</h1>
 						<a class="c-menu__link" href="#">fantasy</a>
 					</li>
 				</ul>
+				<div class="c-select">
+					<label class="c-select__label">
```

Comment:
you can create **label** as a root element with the class "c-select", you don't need **div** wrapper